src/include/sof/samples/audio/c_models_inc/config.h

```diff
@@ -0,0 +1,37 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright 2021 NXP
+ *
+ * Author: Mihai Despotovici <mihai.despotovici@nxp.com>
+ */
+
+#ifndef __CONFIG_H__
+#define __CONFIG_H__
+
+
+// RAW AUDIO DATA CONFIGURATION
+#define MS_TO_SAMPLES(SAMPLERATE, TIME) ((SAMPLERATE) * (TIME) / 1000)
+#define SIZE_FROM_STRIDE_SIZE(STRIDE, SIZE, NO_WINDOWS) ((NO_WINDOWS - 1) * STRIDE + SIZE)
+
```

Comment:
usually the macro parameters should be lower case. It is easier to read:
e.g
```
#define KWD_NN_MS_TO_SAMPLES(sample_rate, time)   ((sample_rate) * (time) / 1000)

```

Reply:
Btw, I think we already have utility functions for these macros.